### PR TITLE
Update dependencies

### LINKED ISSUE #16: Add build tags for tests
See https://github.com/elastic/beats/pull/1073 

This is needed when updated to the most recent version of libbeat.

### LINKED ISSUE #23: Document that glide is required to setiü ürpkect
Details on how setup glide can be found here: https://github.com/Masterminds/glide

### PR DESCRIPTION
**Pull request by ruflin**
* Update Golang to 1.6.0
* Update glide to 0.10.0
* Update libbeat to most recent version
* Update docs about glide requirement
* Add docs about docker requirement to build packages
* Add packaging to test
* Add requirements for gopath in README
* Example test tag for config_test.go added

Closes #16 #23 #29

### CODE REVIEW

**Comment by ruflin** [comment]
Even though the build is green, the yaml update is failing: https://travis-ci.org/elastic/beat-generator/builds/118961853#L340 An issue in the beats repo was opened to solve this issue: https://github.com/elastic/beats/issues/1244

This PR can still be merged.

File: {{cookiecutter.beat}}/README.md [hunk 1]
```diff
@@ -71,6 +76,11 @@ The test coverage is reported in the folder `./build/coverage/`
 
 ### Package
 
+To be able to package {{cookiecutter.beat|capitalize}} the requirements are as following:
```
Comment (andrewkroh) [inline {{cookiecutter.beat}}/README.md]: `s/following/follows/`

**Comment by ruflin** [comment]
Updated and rebased.

File: {{cookiecutter.beat}}/README.md (hunk 1 above)
Comment (ruflin) [inline {{cookiecutter.beat}}/README.md]: fixed

**Pull request merged**